### PR TITLE
EZP-30837: Migrate from repository-forms to ezplatform-content-forms package

### PR DESCRIPTION
**Pull request by webhdx**
> JIRA: https://jira.ez.no/browse/EZP-30837

# Description

This PR removes support for:
* Roles
* Policies
* Limitations
* Sections
* Languages
* Content Types
* User Registration

These parts wasn't mostly used by eZ Platform. Content Types are being moved to AdminUI: https://github.com/ezsystems/ezplatform-admin-ui/pull/1121. User Registration was already deprecated as it's a part of `ezplatform-user` package.

# TODO
- [x] Place link to AdminUI pull request

### CODE REVIEW

File: bundle/Controller/ContentEditController.php [hunk 1]
```diff
@@ -68,11 +68,10 @@ public function createWithoutDraftSuccessAction(ContentCreateSuccessView $view):
     /**
      * Displays a draft creation form that creates a content draft from an existing content.
      *
+     * @param \Symfony\Component\HttpFoundation\Request $request
      * @param mixed $contentId
```
Comment (mikadamczyk) [inline bundle/Controller/ContentEditController.php]: `int|null`

File: lib/FieldType/Mapper/SelectionFormMapper.php [hunk 2]
```diff
@@ -9,86 +9,15 @@
...
-        }
-
-        $fieldDefinitionForm->add(
-            $optionField
-                ->setAutoInitialize(false)
-                ->getForm()
-        );
-    }
-
     public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
```
Comment (mikadamczyk) [inline lib/FieldType/Mapper/SelectionFormMapper.php]: This should be commented?

File: src/lib/FieldType/Mapper/AuthorFormMapper.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+use EzSystems\EzPlatformContentForms\Form\Type\FieldType\AuthorFieldType;
+use Symfony\Component\Form\FormInterface;
+
+/**
+ * FormMapper for ezauthor FieldType.
+ */
+class AuthorFormMapper implements FieldValueFormMapperInterface
+{
+    /**
+    /**
```
Comment (ViniTou) [inline src/lib/FieldType/Mapper/AuthorFormMapper.php]: double doc opening

File: src/lib/Tests/FieldType/FieldTypeFormMapperDispatcherTest.php [hunk 4]
```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+
```
Comment (ViniTou) [inline src/lib/Tests/FieldType/FieldTypeFormMapperDispatcherTest.php]: lines

File: src/lib/Tests/FieldType/FieldTypeFormMapperDispatcherTest.php [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+use EzSystems\EzPlatformContentForms\Data\Content\FieldData;
+use EzSystems\EzPlatformContentForms\FieldType\FieldTypeFormMapperDispatcher;
+use EzSystems\EzPlatformContentForms\FieldType\FieldTypeFormMapperDispatcherInterface;
+use EzSystems\EzPlatformContentForms\FieldType\FieldValueFormMapperInterface;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Form\FormInterface;
+
+class FieldTypeFormMapperDispatcherTest extends TestCase
+{
+    /**
```
Comment (ViniTou) [inline src/lib/Tests/FieldType/FieldTypeFormMapperDispatcherTest.php]: FQCN and single line, dont know if this is fixer fault but those should be checked?

File: .php_cs [hunk 6]
```diff
@@ -1,33 +1,43 @@
...
-        'array_syntax' => false,
+        'array_syntax' => ['syntax' => 'short'],
         'simplified_null_return' => false,
         'phpdoc_align' => false,
-        'phpdoc_separation' => false,
         'phpdoc_to_comment' => false,
         'cast_spaces' => false,
         'blank_line_after_opening_tag' => false,
         'single_blank_line_before_namespace' => false,
-        'phpdoc_annotation_without_dot' => false,
```
Comment (ViniTou) [inline .php_cs]: I think that one is now not consisted across our repositories, why this have changed?

**Comment by ViniTou** [comment]
Is there any way that we could add fixer as separate PR and job to travis and then restart this one? Or can we add it here and use config from this PR?
@mnocon

File: src/lib/FieldType/DataTransformer/TimeValueTransformer.php [hunk 7]
```diff
@@ -19,7 +19,7 @@
 class TimeValueTransformer implements DataTransformerInterface
 {
     /**
-     * @param Value|mixed $value
+     * @param mixed|Value $value
```
[the remark this answers is not in the document]
Comment (webhdx) [reply src/lib/FieldType/DataTransformer/TimeValueTransformer.php]: Yes this comes from CS Fixer.

**Comment by webhdx** [comment]
@ViniTou @mikadamczyk Fixed reported issues. Also I've added code-style check in #3.

**Pull request merged**